Svenbjorn/whatson
# Feature/itinerary

To test: 

Spin up whatson and go to page "itinerary test" 
ensure the "itinerary" part of this page conforms to requirements from sendistovan:
https://sendistovan.invisionapp.com/console/VFI-Tourism---desktop-ckpjdr19e0hqa010ju83s0gs8/ckpjds45e0i9y01y570rd1mgn/play

The required backend endpoints are uploaded to preview
